README.md

```diff
@@ -1 +1,67 @@
+
+
+### Option 1: with BIT and MS backend servers also run locally 
+1. **IMPORTANT!!! You must setup BIT backend server and postgresql database first before setting up the MS-for-BIT server on the next step**.
+To do this, follow the instruction [here](https://github.com/anitab-org/bridge-in-tech-backend/wiki/BIT-development-environment-setup). Once done, you can proceed to the step 2 below. 
+2. Setup MS-for-BIT server by following the setup instruction for Mentorship Backend [here](https://github.com/anitab-org/mentorship-backend) but using the code base from Maya Treacy's fork repository [ms-backend-server](https://github.com/mtreacy002/mentorship-backend/tree/ms-backend-server) branch. 
+To do this, run the following codes on the terminal after you fork and clone the MS backend repository:
+
+```
+$ git checkout -b bit-ms-backend-server develop
+$ git pull https://github.com/mtreacy002/mentorship-backend.git ms-backend-server
+```
+
+Follow the rest of the setup instructions (providing the environment credentials) mentioned on the [MS README `Run app` section](https://github.com/anitab-org/mentorship-backend). **Note** Notice that since **BIT** already occupies **port 5000** of your localhost, the **MS** server is set to run on **port 4000** for this BridgeInTech project.
+
+### Option 2: with BIT and MS backend servers run remotely on Heroku
+
+1. Go to `src/config.js` file 
+2. Comment out line 1 that sets `localhost` as the base REST API url for MS server
+3. Uncomment line 2 to set the [BIT Heroku server](https://bridgeintech-bit-heroku-psql.herokuapp.com) as the base REST url for MS server
+
+<img width="1348" alt="Screen Shot 2020-07-19 at 12 05 25 pm" src="https://user-images.githubusercontent.com/29667122/87865402-3e9f1580-c9b8-11ea-97e1-6dc24fdc970f.png">
+
+**Important**
+Please be aware that there are **drawbacks** if you chose to run the BIT and MS backend servers remotely (**option 2**):
+* There will be a **significant delay** from the time when the request is made on the BIT web GUI to the time the response is received. This is because BIT Heroku makes calls to MS-for-BIT Heroku backend server for MS related functionalities. Both of these Heroku servers are running on [free `dynos`](https://www.heroku.com/dynos) that have limited capacity. When you run both BIT and MS-for-BIT servers locally, you will not have this time lag issue.
+* The BIT Heroku baackend server will only carries the latest code that have been approved and merged to the develop branch. This means, if you are working on a PR or testing an open PR, this **Option 2** (running remote BIT Heroku server) **MUST NOT BE USED**.     
+
+## Run the app
+
+Now that you have set the MS backend, BIT backend and frontend servers, you can run BIT React application by running `npm start`. 
```

Comment:
Can a contributor run `npm start` before `npm install`?
Isn't `npm install` for installing dependencies on web proj? @mtreacy002

Reply:
@isabelcosta , I don't remember if I have ever tried running the frontend before installing the dependencies. But from running frontend through this project, I notice if there's any dependency missing, the app might not render well (or might not render at all). Perhaps similar to Swagger UI often won't render if there's any dependencies missing. That's why I place the npm start after all the backend/frontend setup done.

Reply:
All I am saying is that the `npm install` step is missing. And if could put a 3rd step with the localhost link, people have to go to, that would link to the page. Also a numbered list is easier to read :)

Reply:
Done. Thanks for pointing that out, @isabelcosta . I've made the changes and have pushed the code again. Could you please approve it now? Thanks